Source/MaestroAdvanceAvg.cpp

```diff
@@ -631,16 +658,23 @@ void Maestro::AdvanceTimeStepAverage(bool is_initIter) {
     is_predictor = false;
     AdvancePremac(umac, w0mac_dummy, w0_force_cart_dummy);
 
-    if (evolve_base_state && !split_projection) {
-        Sbar.copy(1.0 / (gamma1bar_nph * p0_nph) * (p0_new - p0_old) / dt);
+    // For the planar case, calculate w0 just before the projection
+    if (evolve_base_state && !spherical) {
+        if (split_projection) {
+            is_predictor = true;
```

Comment:
```suggestion
            is_predictor = false;
```

Reply:
Yes, agree. I had only moved that from where it was before and not considered what it does. I think it should always use `gamma1bar_new_arr` in `Makew0Planar` for this scheme.